docs/DeveloperGuide.md

```diff
@@ -1259,6 +1273,8 @@ Test Cases:
 1. Test case: `search`<br>
    Expected: No module information is displayed. _Invalid command format_ message is shown in the result display.
 
+1. Other invalid search commands to try: `search c/cs2103t`, `search 1`<br>
+   Expected: No module information is displayed. _Invalid command format_ message is shown in the result display.
```

Comment:
for this portion, I think the error message shown is "Invalid module code" instead of "invalid common format"

Reply:
Oh good find!